x-pack/plugins/security_solution/server/lib/detection_engine/signals/rule_status_service.ts

```diff
@@ -93,6 +101,18 @@ export const ruleStatusServiceFactory = async ({
       });
     },
 
+    partialFailure: async (message, attributes) => {
```

Comment:
Is the plan to use this function in `signal_rule_alert_type.ts` or is it already being used somewhere else?

Reply:
Currently, there's no place in the code where this function is being used. When this gets merged I will be incorporating it into the multiple timestamps pr https://github.com/elastic/kibana/pull/83134 I think by returning an error type instead of relying on the `result.success` boolean to write an error / success status.